src/components/TextInput/TextInput.jsx

```diff
@@ -24,6 +25,7 @@ const TextInput = ({
   id,
   inputMask,
   isDisabled,
+  error,
```

Comment:
`hasError`?

Reply:
@JCarmine @juanfabrega I left off the prefix for error because it's not just a boolean. It also accepts a string that when defined will make the input appear red.